nvim-lsp.vim

```diff
@@ -31,16 +31,20 @@ nnoremap <silent> <leader>ca  <cmd>lua vim.lsp.buf.code_action()<CR>
 " nvim-lsp Settings
 "-----------------------------------------------------------------------------
 " If you just use the latest stable version, then this setting isn't necessary
-let g:metals_server_version = '0.9.0+18-27d4652a-SNAPSHOT'
+let g:metals_server_version = '0.9.0+236-8d0924af-SNAPSHOT'
 
 "-----------------------------------------------------------------------------
 " lua callbacks
 "-----------------------------------------------------------------------------
 :lua << EOF
   local nvim_lsp = require'nvim_lsp'
   local metals   = require'metals'
+  local setup    = require'metals.setup'
   local M        = {}
 
+  -- Decoration color. Choose from :highlight
+  metals.decoration.color = 'Conceal'
```

Comment:
See my note above, but I think it's better to instead expose this as a global variable instead of through the lua module.